src/models/todo.model.ts

```diff
@@ -0,0 +1,9 @@
+export type TodoStatus = 'backlog' | 'in-progress' | 'done' | '';
+
+export class Todo {
```

Comment:
For what?

Reply:
I guess, you wanted to use Interface. Try to avoid creating classes in such cases. You would also create a folder called 'interfaces' and a filename would simply be `todo.ts`.